src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1485,15 +1565,21 @@ private static VersionInfo GetVersionInfo()
 
         private readonly IHost _host;
 
+        /// <summary>
+        /// Slope value for this calibrator.
```

Comment:
>Slope value for this calibrator. [](start = 12, length = 32)

Maybe use the phrases `bias` and `weight` instead of `Slope` and `Offset`? This is more consistent with our public API.